nav2_core/include/nav2_core/behavior_tree_navigator.hpp

```diff
@@ -201,13 +201,17 @@ class BehaviorTreeNavigator : public NavigatorBase
     // get the default behavior tree for this navigator
     std::string default_bt_xml_filename = getDefaultBTFilepath(parent_node);
 
+    auto search_directories = node->declare_or_get_parameter("bt_search_directories",
+        std::vector<std::string>{});
```

Comment:
Can we default this to the bt navigator's directory? See the Navigate To Pose Navigator for an example of getting the path. That way we don't require the parameter in the yaml (but we should still keep it). It will help with users' forward porting existing yaml files if they don't explicitly need to include it.